frontend/src/components/Bill/Bill.jsx

```diff
@@ -20,34 +30,180 @@ function getSponsorNumber(billData) {
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      const data = await response.json();
+      if (data.features.length > 0) {
+        const { center } = data.features[0];
+        setLocation({ latitude: center[1], longitude: center[0] });
+        setRepsData([]);
+        setError(null);
+        debouncedFetchReps(center[1], center[0]);
+      } else {
+        setLocation(null);
+        setError('Location not found');
+      }
+    } catch (error) {
+      console.error('Error fetching location:', error);
+      setError('Error fetching location');
+    }
+  };
+
+  if (!bill) return <div>Loading...</div>;
+
+  const {
+    title,
+    summary,
+    sponsor: {
+      member: { fullName: sponsorName },
+    },
+  } = bill;
 
   return (
-    <div>
-      <div>This is the bill {printNo}&apos;s page</div>
-      <h2>{title}</h2>
-      <p>{summary}</p>
-      <div><span>{sponsorName}:{sponsorId}</span></div>
-      <div><span>Total Sponsors: {getSponsorNumber(bill)}</span></div>
-      <RelatedBills related={bill.billInfoRefs.items} />
+    <div className="bill-content">
+      <div className="summary">
+        <h2>{title}</h2>
+        <p>{summary}</p>
+      
+        <div className="category">
+          {bill.category} <p>Sample Category</p>
+        </div><br />
+        <p>
+          Sponsored by <span style={{fontWeight:'bold'}}>{sponsorName}</span>
+          <br />
+          District {bill.sponsor.member.districtCode}
+        </p>
+        <div><p>Total Sponsors: {getSponsorNumber(bill)}</p></div>
+        <p>Related bills:</p>
+        <p><RelatedBills related={bill.billInfoRefs.items} /></p>
+        <p>Status: <span style={{fontWeight:'bold'}}>{bill.status.statusDesc}</span></p>
+        <div className="important">
+          <h4>Why is this important? Why should this bill pass?</h4>
+          <p>
+            Sample reasons why it should be passed!
+            <ol>
+              <li>Reason 1</li>
+              <li>Just because</li>
+            </ol>
+          </p>
+        </div>
+      </div>
+      
+      <div className="action">
+        <h2>Take Action!</h2>
+        <div>
+          <form onSubmit={handleAddressSubmit}>
+            <input
+              type="text"
+              placeholder="Enter full address:"
+              value={query}
+              onChange={handleAddressChange}
+            />
+            <button type="submit">See my reps!</button>
+          </form>
+          {loading && <p>Loading representatives...</p>}
+          {error && <p>{error}</p>}
+          <div>
+            <h3>Your Representatives:</h3>
+            {repsData
+              .filter(rep => rep.jurisdiction.classification !== "country")
+              .map(rep => (
+                <div key={rep.id}>
+                  <h4>{rep.name} - {rep.current_role.title}</h4>
+                  <img src={rep.image} alt={rep.name} style={{ maxWidth: "200px" }} />
```

Comment:
The images slow down the page, and they are inconsistent sizes. I think we should omit them.

Reply:
removed! I think it looked nice, but agree that it slows it down, good idea!

Reply:
There are specific member pages (not fleshed out). We could put images there.